### DISCUSSION TITLE
参考文献格式

### DISCUSSION THREAD

**Discussion by Fanyoupeng-2333** [qianchd/NKU_thesis_graduate#7] [一般化的问题]
请问参考文献默认是平齐的，如何修改为上标形式

**Comment by qianchd** [comment]
使用biblatex宏包，可以用命令`\supercite`

使用natbib宏包，可以在宏包选项里加选项super, 然后再`renewcommand`之类的，可以参考[比如](https://tex.stackexchange.com/questions/420370/how-can-i-use-natbib-to-get-both-superscript-style-citations-and-authoryear-st)

**Comment by Fanyoupeng-2333** [comment]
您好，想请问一下两个问题：（1）参考文献没有显示[J] [c] 这种应该怎么处理呢；（2）citecolor=blue的时候目录也会改变颜色，有什么办法只改变参考文献和图表的颜色，不改变目录颜色吗

**Comment by qianchd** [comment]
涉及样式的问题，
使用biblatex宏包， `\usepackage[backend = bibtex8, defernumbers = true,  sorting=nty,  style = nkthesis]{biblatex}` 一句中是`style = nkthesis`是默认样式，你想要的可能是`style=gb7714-2015`; 如果编译有问题可以试`backend = biber` 之类的别的backend，没测试过这个style

使用natbib宏包, 这个目录里也有几个gb7714开头的bst文件，可以在\bibliographystyle{*} 里填这几个bst的文件名试试看，或者看下怎么在natbib里用gb7714样式在

**Comment by Fanyoupeng-2333** [comment]
请问 生成的pdf，没有参考文献，致谢，个人简历的超链接怎么办

**Comment by Fanyoupeng-2333** [comment]
请问 目录 二级标题和三级标题字体大小怎么调整

**Comment by qianchd** [comment]
> 请问 生成的pdf，没有参考文献，致谢，个人简历的超链接怎么办

没懂，目录里没有？看看有没有报错信息，重复多编译几次，一般都能解决问题。没遇到过类似的问题。不行的话需要自己研究一下hyperref的宏包。

> 请问 目录 二级标题和三级标题字体大小怎么调整

在`NKThesis.sty` `NKThesis.cfg`  `NKTfonts.cfg` 这些文件里搜一下`section`这个关键词，然后定位一下需要改的代码位置，各种能改的设定基本都在这些里面，找不到就没法改了。原始的设定应该就符合学校要求了，不建议自己改得太复杂。
